test/unit/attributes.js

```diff
@@ -1117,7 +1139,7 @@ QUnit.test( "val(select) after form.reset() (Bug #2551)", function( assert ) {
 } );
 
 QUnit.test( "select.val(space characters) (gh-2978)", function( assert ) {
-	assert.expect( 35 );
+	assert.expect( 37 );
```

Comment:
To be specific, the bug fix cause these tests to fail. Just editing them accomplished the same thing as adding another test to verify the correct behavior. We were verifying buggy behavior before.